clar2wasm/src/wasm_generator.rs

```diff
@@ -1056,14 +1107,15 @@ impl WasmGenerator {
                     // Push the offset and size to the data stack
                     builder.local_get(offset).i32_const(size);
 
-                    // Call the host interface function, `tx_sender`
-                    builder.call(
-                        self.module
-                            .funcs
-                            .by_name("stdlib.tx_sender")
-                            .expect("function not found"),
-                    );
-                    true
+                    builder.call(self.module.funcs.by_name("stdlib.tx_sender").ok_or_else(
```

Comment:
I suggest we change all of these to use the `func_by_name` method. In a separate PR, we can update that method to replace the `panic` with an error and enable the clippy check for panics as well.

Reply:
Alright. If it's not much work, I can create a new PR asap. Changes implemented